recipes/terragrunt/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+go_compiler:
+  - go-nocgo
```

Comment:
this is not needed, we have the go_compiler in conda_build_config.yaml from conda-forge-pinning.